README.md

```diff
@@ -46,8 +46,30 @@ $ npm run download-offence-code-data
 
 # Combine all sources into the final output in output-data/
 $ npm run merge-offence-data
+
+```
+###  Accessing the PNLD website
+
+We use puppeteer to interact with the PNLD in the browser. From time to time we experience differences in the browser through updates to html or broken links so its helpful debugging. To access it in the browser go to [PNLD](https://www.pnld.co.uk/standard-offence-wording-extracts/) and look in the table bellow to find out where to get the credentials 
+
+The function `PnldFileDownloader` is called by the script `download-offence-code-data.ts` and needs four environment variables in order to access the PNLD service.
+
+| environment variable    | description        |
+| ----------------------- | ----------------------------------------------------------- |
+| PNLD_USERNAME           | User name can be found in 1password in the shared vault     |
+| PNLD_PASSWORD           | Password can be found in 1password in the shared vault      |
+| PNLD_LOGIN_URL          | https://www.pnld.co.uk/standard-offence-wording-extracts/   |
+| PNLD_DOWNLOAD_URL       | This depends on the zip file we want to download: <br> Full extract: https://www.pnld.co.uk/standard-offence-wording-extracts/full-extract <br>Monthly Update (Current Month): https://www.pnld.co.uk/standard-offence-wording-extracts/monthly-delta-extract<br>Monthly Updates (Last Month): https://www.pnld.co.uk/standard-offence-wording-extracts/monthly-delta-extract-1-month-prior), [Monthly Updates (2 Months Ago): https://www.pnld.co.uk/standard-offence-wording-extracts/monthly-delta-extract-2-months-prior
```

Comment:
Is it possible to make the types of zip files a bulleted list within the table description? 🤔

I think we have an stray `[` btw on "Monthly Updates(2 months ago)!